src/ChronosDate.php

```diff
@@ -90,35 +91,92 @@ class ChronosDate
      * subtraction/addition to have deterministic results.
      *
      * @param \Cake\Chronos\Chronos|\Cake\Chronos\ChronosDate|\DateTimeInterface|string $time Fixed or relative time
+     * @param \DateTimeZone|string|null $timezone The time zone used for 'now'
      */
-    public function __construct(Chronos|ChronosDate|DateTimeInterface|string $time)
-    {
-        $this->native = $this->createNative($time);
+    public function __construct(
+        Chronos|ChronosDate|DateTimeInterface|string $time = 'now',
+        DateTimeZone|string|null $timezone = null
+    ) {
+        $this->native = $this->createNative($time, $timezone);
     }
 
     /**
      * Initializes the PHP DateTimeImmutable object.
      *
-     * @param \Cake\Chronos\Chronos|\Cake\Chronos\ChronosDate|\DateTimeInterface|string|int|null $time Fixed or relative time
+     * @param \Cake\Chronos\Chronos|\Cake\Chronos\ChronosDate|\DateTimeInterface|string $time Fixed or relative time
+     * @param \DateTimeZone|string|null $timezone The time zone used for 'now'
      * @return \DateTimeImmutable
      */
-    protected function createNative(Chronos|ChronosDate|DateTimeInterface|string|int|null $time): DateTimeImmutable
-    {
+    protected function createNative(
+        Chronos|ChronosDate|DateTimeInterface|string $time,
+        DateTimeZone|string|null $timezone
+    ): DateTimeImmutable {
+        if (!is_string($time)) {
+            return new DateTimeImmutable($time->format('Y-m-d 00:00:00'));
```

Comment:
Should this use the timezone parameter?

Reply:
No, the time zone is only for interpreting the time string or now.

Reply:
Ok. I was concerned that we'll be shifting timezones to server default from the provided datetime. Thinking about it more though what you have is correct as we're preserving the date aspect and timezones are only relevant when figuring out relative concepts like today/yesterday/tomorrow.